vm/mx.vm/mx_vm.py

```diff
@@ -578,10 +631,28 @@ def native_image(self, build_args, output_file, allow_server=False, nonZeroIsFat
         name = basename(output_file)
         if suffix:
             name = name[:-len(suffix)]
+
         native_image_command += [
             '-H:Path=' + output_directory or ".",
             '-H:Name=' + name,
         ]
+
+        if _get_jdk().version.parts[0] >= 11:
+            graal_sdk_jar = join(stage1.output, stage1.find_single_source_location('dependency:sdk:GRAAL_SDK'));
```

Comment:
I'm surprised this is needed, i would expect the code guarded by `useJavaModules()` in `NativeImage.java` to already take care of whatever is necessary.
/cc @olpaw

Reply:
I agree. This seems redundant/misguided. If any changes are necessary to build images with Java 11 they should be added to the already existing Java 11 support in native image. See:
https://github.com/oracle/graal/blob/af03ea82d35937f6d7bea8ec265ace6d88fffb94/substratevm/mx.substratevm/mx_substratevm.py#L138 (all definitions of `GRAAL_COMPILER_FLAGS_MAP['11']`)
https://github.com/oracle/graal/blob/af03ea82d35937f6d7bea8ec265ace6d88fffb94/substratevm/mx.substratevm/mx_substratevm.py#L1161
https://github.com/oracle/graal/blob/dde3d6fdd57256958b22621bbe74fd379507924c/substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java#L117
https://github.com/oracle/graal/blob/dde3d6fdd57256958b22621bbe74fd379507924c/substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java#L294
https://github.com/oracle/graal/blob/dde3d6fdd57256958b22621bbe74fd379507924c/substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java#L223 (and search for all uses of `useJavaModules()`)